extensions/blocks/publicize/twitter/editor.scss

```diff
@@ -0,0 +1,121 @@
+h3.jetpack-publicize-twitter-options__heading {
+	margin-top: 1em;
+}
+
+.jetpack-publicize-twitter-options__preview {
+	width: 100%;
+	justify-content: center;
+}
+
+.jetpack-publicize-twitter__tweet-divider {
+	position: absolute;
+	width: 100%;
+	margin-top: -28px;
+
+	.jetpack-publicize-twitter__tweet-divider-icon {
+		margin: 0 auto;
+		display: block;
+		background: #0009;
```

Comment:
Nope, the use of black-with-transparency (and white-with-transparency for dark themes) is intentional: the UI needs to be visible against any background colour the theme might set in the editor. Gutenberg itself handles this by providing starkly contrasting UI: it's all black and white, with appropriate borders for each. The Twitter UI is intended to be more subtle, so that it can remain visible, without distracting from the content itself.

For interest, I wrote up an experiment to evaluate the accessibility and visual appearance of different backgrounds against a large range of background colours.

https://jsfiddle.net/pento/83kd0na5/2/